### PR TITLE
Add feature to install beta skills

### PR DESCRIPTION
**Pull request by MatthewScholefield**
This relates to [this mycroft-core PR](https://github.com/MycroftAI/mycroft-core/pull/1747) and adds in the capability of installing skills as beta. The following are examples of possible queries:
 - `install mycroft timer beta` - upgrade or install the beta version
 - `install mycroft timer stable` - downgrade to stable or install the stable version
 - `install mycroft timer` - same as above

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -36,27 +36,93 @@ def initialize(self):
...
+            skill_data['beta'] = False
+
+            if not was_beta and skill.is_local:
                 raise AlreadyInstalled(skill.name)
 
-        self.confirmable_interaction(
-            SkillEntry.install, False, check_installed, message.data,
-            self.install_word, 'install.confirm', 'install.complete'
-        )
+            self.write_skills_data(skills_data)
```
Comment (forslund) [inline __init__.py]: Should the beta value be set to false and skill data be written before the action is confirmed?

I believe I got it to switch the beta flag to false and keep the beta version by not replying to the confirmation

File: __init__.py [hunk 2]
```diff
@@ -36,27 +36,93 @@ def initialize(self):
...
+            skill_data = skills_data.setdefault(skill.name, {})
+            skill_data['manual_install'] = True
+            was_beta = skill_data.get('beta')
+            skill_data['beta'] = True
+
+            if was_beta and skill.is_local:
+                self.speak_dialog('error.already.beta', dict(skill=skill.name))
+                return
+
+            self.write_skills_data(skills_data)
```
Comment (forslund) [inline __init__.py]: Same here possibly?

File: __init__.py (hunk 1 above)
Comment (MatthewScholefield) [reply __init__.py]: Yep! Nice catch!

**Review by forslund — APPROVED** [review]
Looks good now

**Pull request merged**